devops/deploy-on-lightsail.yml

```diff
@@ -2,7 +2,7 @@
 - name: deploy latest new train tracker on lightsail
   hosts: all
   remote_user: ubuntu
-  become: yes
+  become_user: ubuntu
```

Comment:
Pretty sure this is why `root` started owning lots of things; `remote_user` is the user to ssh in as, and then `become: yes` was escalating to root